Cubical/Categories/Functor/Properties.agda

```diff
@@ -58,6 +58,13 @@ F-assoc {F = F} {G} {H} i .F-hom f = H ⟪ G ⟪ F ⟪ f ⟫ ⟫ ⟫
 F-assoc {F = F} {G} {H} i .F-id {x} =  congAssoc (G ⟪_⟫) (H ⟪_⟫) (F .F-id {x}) (G .F-id {F ⟅ x ⟆}) (H .F-id) (~ i)
 F-assoc {F = F} {G} {H} i .F-seq f g =  congAssoc (G ⟪_⟫) (H ⟪_⟫) (F .F-seq f g) (G .F-seq _ _) (H .F-seq _ _) (~ i)
 
+-- is the above version in any way preferable?
+-- think this computes better:
```

Comment:
Wow, I think this proof looks substantially better! @ecavallo or @felixwellen  Should we just delete the old complicated proof?

Reply:
I just deleted the old proof now